### DISCUSSION TITLE
Launching test app correctly

### DISCUSSION THREAD

**Discussion by Ruandup90** [reflex-dev/reflex#3796] [Q&A] [answered]
Good day community, 

Hope you are well! 

I have recently started with python web app development and came across reflex - what a cool product! I am trying to use the customer data app as an example and cloned the code from Github (https://github.com/reflex-dev/reflex-examples/tree/main/customer_data_app)

I opened the folder in VS Code and tried to run the python file in the terminal. At first, it gave errors regarding referencing modules in other directories. I thought I would cross that bridge by placing all the required modules in the same file so no referencing is needed and that seemed to work since there are no errors / problems in the code. Here is the python code (just in txt file version) for reference: [customer_data_app_v2.txt](https://github.com/user-attachments/files/16624364/customer_data_app_v2.txt)

My problem is when I execute the code in the terminal, no output is created... it just shows that the code executed but that is it. Any idea how I can "activate" the output? I hope to end up with something like "your session is active at this URL: ..." which I can launch? 

Thanks in advance for your help!

**Comment by picklelo** [comment, marked as answer]
Hey! Are you doing `reflex run`? What errors do you get?

**Reply by Ruandup90** [reply]
Thank you so much picklelo, I accidently ran "python " and the py code reference... Thanks man, it is running lekker now!
